test/pairwise.jl

```diff
@@ -0,0 +1,32 @@
+using Test
+using DFTK
+using LinearAlgebra
+using Random
+
+@testset "Pairwise forces" begin
+    a = 5.131570667152971
+    lattice = a .* [0 1 1; 1 0 1; 1 1 0]
+    # perturb positions away from equilibrium to get nonzero force
+    positions = [ones(3)/8+rand(3)/20, -ones(3)/8]
+    charges = [:Si, :Si]
+    atom_types = map(x -> ElementCoulomb(x).symbol, charges)
```

Comment:
This is a void statement as `atom_types == charges`. Just remove both of them and put the respective values in-line into the dict.

Reply:
I still need atom_types, but removed charges.